onadata/libs/utils/export_builder.py

```diff
@@ -1015,8 +1015,13 @@ def _get_sav_value_labels(self, xpath_var_names=None):
                                 if (float(name) > int(name)) else int(name)
                         except ValueError:
                             pass
-                    label = self.get_choice_label_from_dict(choice['label'])
-                    _value_labels[name] = label.strip()
+                    try:
+                        label = self.get_choice_label_from_dict(
```

Comment:
Could this be a less verbose fix? 
`choice.get("label", "")`

Reply:
While this would look neater, I believe that we would be making an assumption that the form's choices would always be a dictionary with the following syntax:

`{'name': 'red', 'label': 'red'}`

This does not cover instances where the form supports multiple languages.
After investigating, I realized the `get_choice_label_from_dict` function unpacks the label attribute in an instance where the choices obtained would yield a dict of this syntax:

`{'name': 'orange', 'label': {'Swahili': 'Chungwa', ' English': 'Orange'}}`